src/main/java/org/example/bookshop/model/Book.java

```diff
@@ -38,6 +45,16 @@ public class Book {
     @Column(nullable = false)
     private BigDecimal price;
 
+    @ManyToMany
+    @JoinTable(
+            name = "books_categories",
+            joinColumns = @JoinColumn(name = "book_id"),
+            inverseJoinColumns = @JoinColumn(name = "categories_id")
+    )
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
```

Comment:
you don't have such methods but you can leave it

Reply:
Don't forget to exclude entity fields from toString(), equals(), and hashCode() methods. Example: ....

This was a condition of the task